README.md

```diff
@@ -82,7 +90,11 @@ Write the BNF (not EBNF) description for this language.
 
 ## A2.1
 
-«replace this with your answer»
+< adjective >   ::=     ""|< a >|< adjective > < a >  
+< a >           ::=     "lazy"|"smelly"  
+< noun >        ::=     "dog"|"cat"  
+< verb >        ::=     "ate"|"ran"  
+< adverb >      ::=     ""|"slowly"|"noisily"  
```

Comment:
# 4

There doesn't seem to be a sentence here.

Also, 

    < adjective >   ::=     ""|< a >|< adjective > < a >  

is actually ambiguous, because <adjective> could be "", so the second and third clauses on the RHS could both match. Better would be
  
    < adjective >   ::=     "" | < adjective > < a >